src/AlgebraicGeometry/RationalPoint/ProjectiveRationalPoint.jl

```diff
@@ -123,7 +139,12 @@ end
 
 function is_smooth_at(X::AbsProjectiveScheme{<:Field}, P::AbsProjectiveRationalPoint)
   @req P in X "not a point on X"
-  error("not implemented")
+  X = codomain(P)
+  S = standard_covering(X)
+  n = length(coordinates(P))
+  i = findfirst(i->!iszero(P[i]),1:n)
```

Comment:
Could perhaps also do this? (Feel free to ignore)
```suggestion
  i = findfirst(!iszero,coordinates(P))
```

Reply:
thanks for the hint